fix : error from labels with slash "/" while searching for issues

awaismemon01 opened the pull request:
description : Labels with slash "/" threw exception because of security mechanism of tomcat.

awaismemon01 committed:
fix : error from labels with slash

description : Labels with slash "/" threw exception because of security mechanism of tomcat.

awaismemon01 committed:
feat : add user and query for persistence layer

awaismemon01 committed:
fix : sonar security issue

ifourtechnolab commented:
make smaller commits

ifourtechnolab commented on src/main/java/io/polyrepo/analyser/PolyRepoAnalyserApplication.java:
```diff
@@ -3,15 +3,24 @@
 @EnableSwagger2
 @EnableFeignClients
-public class PolyRepoAnalyserApplication {
+public class PolyRepoAnalyserApplication implements WebMvcConfigurer {
```
understand what this interface is doing

ifourtechnolab commented on pom.xml:
```diff
@@ -76,6 +76,17 @@
+        <!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-jdbc -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jdbc</artifactId>
```
use simple JDBC

ifourtechnolab commented on src/main/java/io/polyrepo/analyser/repository/IQueryRepository.java:
```diff
@@ -0,0 +1,13 @@
+
+
+@Repository
+public interface IQueryRepository {
```
Use convention as interface name and then add `impl` to implementing class

awaismemon01 committed:
refactor : naming convention of interface and impl class

description : remove 'I' from interface name , add 'Impl' as suffix in implementing class.
